packages/block-library/src/image/edit.js

```diff
@@ -512,6 +541,45 @@ class ImageEdit extends Component {
 								}
 							}
 							/* eslint-enable no-lonely-if */
+							if ( ! alt ) {
+								return (
```

Comment:
There's a ton of code duplication here. What's actually different? Can we refactor to limit the variation of this specific condition? i.e. if it's to return `imgNoAlt` instead of `img`, just change that in the existing return value:

```
{ alt ? img : imgNoAlt }
```